assignment1.txt

```diff
@@ -0,0 +1,11 @@
+Ashank Kumar Assignment 1
+
+1. A compiler is a program that converts code written in one programming language such as C into another proggramming language, such as assembly.
```

Comment:
sort of, more specifically, a compiler takes your code written in C (a higher level language), removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.